gtk2.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
can you replace this with the gtk2 from shared-modules? https://github.com/flathub/shared-modules/ as a git submodule

Reply:
On IRC I was told to copy it :/

Reply:
He said feel free to copy how other applications use it, not copy the shared-modules.